sbin/prepareWorkspace.sh

```diff
@@ -577,15 +577,33 @@ prepareMozillaCacerts() {
     fi
 }
 
-# Download all of the dependencies for OpenJDK (Alsa, FreeType etc.)
+downloadBootJdkIfNeeded () {
+  if [[ "${BUILD_CONFIG[JDK_BOOT_DIR]}" == "download" ]]; then
+    # the bootDir is used by refactored downloaders.sh methods; it would benice to change in future
```

Comment:
```suggestion
    # The bootDir is used by downloaders.sh methods; it would be nice to change in future
```

Reply:
Change to what?

Reply:
`    # The bootDir is used by downloaders.sh methods; it would be nice to change it in future so the download method is unified`

Sounds better?